src/pages/Entries/HappinessCard.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import Card from "../../components/Card";
+import { Happiness } from "../../data/models/Happiness";
+
+export default function HappinessCard({
+  data,
+  selected,
+  click,
+}: {
+  data: Happiness;
+  selected: boolean;
+  click: () => void;
+}) {
+  const date = new Date(data.timestamp + "T00:00:00");
+  let classes = "";
+  if (date.toLocaleDateString("sv") === new Date().toLocaleDateString("sv")) {
```

Comment:
I feel like you shouldn't be comparing date values using strings, are you sure there isn't a better way to do this using the JavaScript Date library?

Reply:
I think we have all learned how bad the js date lib is lol